pyspecdata/plot_funcs/pcolormesh.py

```diff
@@ -37,13 +37,16 @@ def pcolormesh(
         because matplotlib doesn't let you call sharex/sharey more than once,
         you need then to tell it not to handle the axis sharing, and to it yourself
         outside this routine.
-    mappable_list: list (default [])
-        empty list which fills with field values from color axis used for initial subplot, used to scale multiple plots along the same color axis. Used to make all 3x2 plots under a uniform color scale
+    mappable_list : list, default []
+        empty list which fills with field values from color axis used for
+        initial subplot, used to scale multiple plots along the same color
+        axis. Used to make all 3x2 plots under a uniform color scale
```

Comment:
again, fixing it to be to spec (see also [here](https://github.com/numpy/numpydoc/issues/284)).

note also that I'm doing `gq` to wrap the text to a reasonable width.